dispatch/api/urls.py

```diff
@@ -20,6 +20,7 @@
 router.register(r'dashboard', views.DashboardViewSet, base_name='api-dashboard')
 router.register(r'integrations', views.IntegrationViewSet, base_name='api-integrations')
 router.register(r'files',views.FileViewSet, base_name='api-files')
+router.register(r'images',views.ImageViewSet, base_name='api-images')
```

Comment:
I think this got carried over from the other branch. router already registers images url on line 17